### PR TITLE
Add a method to check if a single pod running

### PR DESCRIPTION
**Pull request by Fredy-Z**
Sometimes we may not want to check if all pods are running but just one single pod.

So I add one function `WaitForPodRunning` to fulfill this need.

### CODE REVIEW

File: test/kube_checks.go [hunk 1]
```diff
@@ -104,12 +104,32 @@ func WaitForAllPodsRunning(client *KubeClient, namespace string) error {
...
 	for _, pod := range podList.Items {
-		if pod.Status.Phase != corev1.PodRunning && pod.Status.Phase != corev1.PodSucceeded {
+		if isRunning, _ := PodRunning(&pod); !isRunning {
 			return false, nil
 		}
 	}
 	return true, nil
 }
+
+// PodRunning will check the status conditions of the pod and return true if it's Running
```
Comment (vagababov) [inline test/kube_checks.go]: Is this going to be used outside of this file?

File: test/kube_checks.go [hunk 2]
```diff
@@ -104,12 +104,32 @@ func WaitForAllPodsRunning(client *KubeClient, namespace string) error {
...
+func PodRunning(pod *corev1.Pod) (bool, error) {
+	if pod.Status.Phase != corev1.PodRunning && pod.Status.Phase != corev1.PodSucceeded {
```
Comment (vagababov) [inline test/kube_checks.go]: this function never returns error. So just bool.

File: test/kube_checks.go (hunk 1 above)
Comment (vagababov) [reply test/kube_checks.go]: Hm... ignore this one :)

File: test/kube_checks.go (hunk 2 above)
Comment (Fredy-Z) [reply test/kube_checks.go]: Done.

File: test/kube_checks.go [hunk 3]
```diff
@@ -104,12 +104,32 @@ func WaitForAllPodsRunning(client *KubeClient, namespace string) error {
...
+		if isRunning := PodRunning(&pod); !isRunning {
 			return false, nil
 		}
 	}
 	return true, nil
 }
+
+// PodRunning will check the status conditions of the pod and return true if it's Running
+func PodRunning(pod *corev1.Pod) bool {
+	if pod.Status.Phase != corev1.PodRunning && pod.Status.Phase != corev1.PodSucceeded {
```
Comment (vagababov) [inline test/kube_checks.go]: and now we can inline this into: 
`return pod.Status.Phase == corev1.PodRunning || pod.Status.Phase == corev1.PodSucceeded` :-)

Comment (Fredy-Z) [reply test/kube_checks.go]: Done. Thanks!

/meow

**Comment by vagababov** [comment]
That's a huge cat.

**Pull request merged**